Detectors/AOD/include/AODProducerWorkflow/AODProducerHelpers.h

```diff
@@ -55,7 +53,7 @@ auto createTableCursor(framework::ProcessingContext& pc)
   framework::Produces<T> c;
   c.resetCursor(pc.outputs()
                   .make<framework::TableBuilder>(framework::OutputForTable<T>::ref()));
-  c.setLabel(o2::aod::MetadataTrait<T>::metadata::tableLabel());
```

Comment:
Can't we rewrite `aod::MetadataTrait<T>::metadata::tableLabel()` to simply return `o2::aod::Hash<T::ref.label_hash>::str`? This way we would not need to change a few dozens of  lines all at once.

Reply:
Unfortunately no, since the metadata trait is now defined only on the description/version hash and has no information about the label.

Reply:
I do not understand. T is the same class, no?

Reply:
Introduced a helper function that hides the internals.